client/src/components/BrowseChefs/FilterChefs.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from "react";
+import {
+  Box,
+  Typography,
+  FormControl,
+  OutlinedInput,
+  Chip,
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+import LocationOnIcon from "@material-ui/icons/LocationOn";
+
+const FilterChefs = (props) => {
+  const useStyles = makeStyles((theme) => ({
+    searchField: {
+      marginTop: theme.spacing(9),
+      marginBottom: theme.spacing(6),
+    },
+
+    cuisineChip: {
+      borderRadius: "0px",
+    },
+  }));
+
+  const classes = useStyles();
+
+  const cuisineTags = [
+    "ALL",
+    "AMERICAN",
+    "BRITISH",
+    "CARIBBEAN",
+    "CHINESE",
+    "FRENCH",
+    "GREEK",
+    "INDIAN",
+    "ITALIAN",
+    "JAPANESE",
+    "MEDITERRANEAN",
+    "MEXICAN",
+    "MORROCAN",
+    "SPANISH",
+    "THAI",
+    "TURKISH",
+    "VIETNAMESE",
+  ];
+
+  const [activeCuisineTags, setActiveCuisineTags] = useState([cuisineTags[0]]);
+
+  const addActiveCuisineTag = (cuisineTag) => {
+    if (activeCuisineTags.includes(cuisineTag)) return;
+
+    setActiveCuisineTags((activeCuisineTags) => {
```

Comment:
Giving the previous state variable the same name as the state variable is a little confusing (for a second I thought you were mutating state directly), would be more clear to use a name like `prevCuisineTags`

Reply:
I changed it. Thank you. 👍